charts/tekton-apps/README.md

```diff
@@ -112,6 +113,12 @@ spec:
                 pm: xxx
                 tm: xxx
               namespace: prod
+              notifications:
+                annotations:
+                  notifications.argoproj.io/subscribe.on-health-degraded.slack: project-vp; project-vp-ci; project-vp-alarms
+                  notifications.argoproj.io/subscribe.on-sync-failed.slack: project-vp; project-vp-ci; project-vp-alarms
+                  notifications.argoproj.io/subscribe.on-sync-status-unknown.slack: project-vp; project-vp-ci; project-vp-alarms
+                  notifications.argoproj.io/subscribe.on-sync-succeeded.slack: project-vp; project-vp-ci; project-vp-alarms
```

Comment:
Do we need to send notifications to all these channels?
We are supposed to copy-paste this from README, so I'd suggest to adjust it to be a perfect sample - how it is recommended to setup notifications. 

@dmitry-mightydevops what options are preferrable?
Ex. I don't think developers should take care aboyt sync success - that will just "make noize" in main channed and if it is really used as primary communication, then too much automated notifications will drive people to use another channel for "human talks".

I'd suggest something like this:
```yaml
notifications.argoproj.io/subscribe.on-health-degraded.slack: project-vp; project-vp-alarms
notifications.argoproj.io/subscribe.on-sync-failed.slack: project-vp-ci; project-vp-alarms
notifications.argoproj.io/subscribe.on-sync-status-unknown.slack: project-vp-alarms
notifications.argoproj.io/subscribe.on-sync-succeeded.slack: project-vp-ci
```

Reply:
@populov 
I want this specifically to notify devs about issues of the apps they're working on. 

- **on-sync-status-unknown** can be caused by expired bitnami tag of the wordpress and dev team should ping devops engineers to update tekton-apps configs and therefore the dev team should know about it.
- **on-sync-succeeded** - reason we need it is to notify after the degrade is fixed, that the app is OK. We may want to remove it. 

That also works as a signal - deployment is finalized.

So I would vote with the following:



### notifications.argoproj.io/subscribe.on-health-degraded.slack:   
- project-vp; 
- project-vp-alarms

### notifications.argoproj.io/subscribe.on-sync-failed.slack: 
- project-vp-ci
- project-vp-alarms

### notifications.argoproj.io/subscribe.on-sync-status-unknown.slack: 
- project-vp
- project-vp-alarms

### notifications.argoproj.io/subscribe.on-sync-succeeded.slack: 
- project-vp (we can skip this one  if we think that devs don't really need success 2nd notification after the failure. I think we need one)
- project-vp-ci
- project-vp-alarms

@luciano-buono @populov thoughts?

@saritasa-nest/saritasa-devops pls review this PR :)

Reply:
For `degraded` and `sync-failed` like @populov  suggested

For the other two I think is enough:
```
notifications.argoproj.io/subscribe.on-sync-status-unknown.slack: project.vp; project-vp-alarms
notifications.argoproj.io/subscribe.on-sync-succeeded.slack: project-vp-ci; project-vp-alarms
```
is enough

Reply:
ok let's go with that @luciano-buono

Reply:
@luciano-buono Please update README.md and Chart.yaml with defaults you agreed with @dmitry-mightydevops  and @populov and add note for devops team that this how all triggers should be set by default (except team desire for custom triggers config)